src/App.scss

```diff
@@ -110,3 +112,36 @@ card {
   border: 0.9px solid #d4c9c9 !important;
   max-width: 75%;
 }
+
+.btn-light.btn-search {
```

Comment:
These css rules related to the `search` feature is better placed within the `Search` component folder.

Reply:
@JaymesKat 
since I am overriding global CSS from mdbootstrap, when I place these rules local to the component css they always do not work.

Reply:
Ok then, let us have them there